src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
     }
```

Comment:
Avoid creating a new instance of Random every time the method is called. Instead, create a single instance as a class field and reuse it. Additionally, use the 'name()' method instead of 'toString()' to get the String representation of enum constants to ensure the correct name is returned even if 'toString()' is overridden.